news/2024/2024-05-01-the-nat-gazette-june-2024.md

```diff
@@ -0,0 +1,150 @@
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="THE VIDEO LINK. ADD THE VIDEO LINK." frameborder="0" allowfullscreen></iframe>
+</div>
+
+These changes come after an extensive community discussion was held in the [BN Management subforum](https://osu.ppy.sh/community/forums/115). Moving on, big changes like these will be posted there as to foster a healthy discussion with everyone in the community. As the changes to come will come out in cycles, it is very important to get your feedback — your voice matters!
+
+## Updates
+
+This time around we bring quite a sackful of changes. From additions in the ranking criteria, to sweeping changes in the way both new applicants and current BNs will be evaluated, there's a little of something for everyone.
+
+### Evaluations moving forward
+
+Based on recent community feedback, the Nomination Assessment Team will no longer take a proactive approach enforcing arbitrary mapping quality standards in their evaluations. We believe the community as a whole must be the one to decide what's best for the platform.
+
+This change in perspective means that Beatmap Nominators **will not** be punished for any subjective concern that the evaluators may have had on beatmaps that have been ranked without resets. Unrankables, unranks, and any other objective issue will still be handled as per usual.
+
+While we may experiment with more changes depending on how things turn out, the direction of letting the community set their own standards will not change.
+
+### Community-wide changes
+
+#### Beatmap Management subforum
+
+As briefly spoiled before, we have created the brand-new [*BN Management* subforum](https://osu.ppy.sh/community/forums/115). This subforum aims to be a place for any Beatmap Nominator to start a discussion regarding the user group, Ranked section, or any of the systems involved with it. This not only aims to empower BNs by giving them an outlet to express their ideas, but also for the playerbase to interact and discuss in any open post and provide feedback to the proposals.
+
+#### Public evaluations archive
+
+A public repository with both applications and current BN evaluations can now be accessed by anyone! The repository can be found in the [BN website](https://bn.mappersguild.com/publicarchive). The goal of this change was for applicants to have a point of reference and learn from existing applications to know what to expect! Currently, only users who opt in will have their evaluations shown in the repository.
+
+![](/wiki/shared/news/2024-05-01-the-nat-gazette-june-2024/public-eval-archive.jpg)
+
+#### Ranking criteria changes
+
+| Type | Change | Link |
+| :-- | :-- | :-- |
+| **General** | Songs and artists currently disallowed in the [Content usage permissions](/wiki/Rules/Content_usage_permissions#artist-permissions) will **not** be able to be Ranked without explicit approval from their respective owners. | [View changes](https://github.com/ppy/osu-wiki/pull/11262) |
```

Comment:
```suggestion
| **General** | Beatmaps of songs and artists currently disallowed in the [Content usage permissions](/wiki/Rules/Content_usage_permissions#artist-permissions) will **not** be able to be Ranked without explicit approval from their respective owners. | [View changes](https://github.com/ppy/osu-wiki/pull/11262) |
```

Reply:
Think it's fine without needing to clarify it, just like it wasn't [in the RC](https://osu.ppy.sh/wiki/en/Ranking_criteria#tools:~:text=For%20artists%20or%20tracks%20which%20are%20listed%20as%20disallowed%20on%20the%20page%2C%20their%20songs%20may%20only%20be%20ranked%20if%20explicit%20permission%20for%20the%20track%20is%20given%20to%20the%20beatmap%20host.)

`Beatmaps of songs and artists...` sounds a bit weird on the tongue to me, though it may just be me.

Reply:
sure np